src/style/app.module.css

```diff
@@ -1942,185 +2058,794 @@ input[type='radio']:checked + label:hover {
+    width: 100%;
+  }
+}
+
+@media (max-width: 550px) {
+  .orgCard {
+    width: 100%;
+  }
+
+  .orgCard {
+    height: unset;
+    margin: 0.5rem 0;
+    padding: 1.25rem 1.5rem;
+  }
+
+  .orgCard .innerContainer .orgImgContainer {
+    margin-bottom: 0.8rem;
+  }
+
+  .orgCard .innerContainer {
+    flex-direction: column;
+  }
+
+  .orgCard .innerContainer .orgImgContainer img {
+    height: auto;
+    width: 100%;
+  }
+
+  .orgCard .innerContainer .content {
+    margin-left: 0;
+  }
+
+  .orgCard button {
+    bottom: 0;
+    right: 0;
+    position: relative;
+    margin-left: auto;
+    display: block;
+  }
+  .joinBtn,
+  .joinedBtn,
+  .withdrawBtn {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+  }
+}
+
+.colorLight {
+  background-color: #f5f5f5;
+}
+
+.mainContainer50 {
+  width: 50%;
+  flex-grow: 3;
+  padding: 1rem;
+  max-height: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+  background-color: #f2f7ff;
+}
+
+.link {
+  text-decoration: none !important;
+  color: black;
+}
+
+.postInputContainer {
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+}
+
+.maxWidthUserPost {
+  width: 100%;
+}
+
+.postActionContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.postActionBtn {
+  background-color: white;
+  border: none;
+  color: black;
+}
+
+.postActionBtn:hover {
+  background-color: ghostwhite;
+  border: none;
+  color: black;
+}
+
+.postInput {
+  resize: none;
+  border: none;
+  outline: none;
+  box-shadow: none;
+  background-color: white;
+  margin-bottom: 10px;
+}
+
+.postInput:focus {
+  box-shadow: none;
+}
+
+.postContainer {
+  width: auto;
+  background-color: white;
+  margin-top: 1rem;
+  padding: 1rem;
+  border: 1px solid #dddddd;
+  border-radius: 10px;
+}
+
+.heading {
+  font-size: 1.1rem;
+}
+
+.pinnedPostsCardsContainer {
+  overflow-x: scroll;
+  display: flex;
+  gap: 1rem;
+  --bs-gutter-x: 0;
+}
+
+.postsCardsContainer {
+  /* display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  --bs-gutter-x: 0; */
+}
+
+.userImageUserPost {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  margin-left: 1rem;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  border-radius: 50%;
+  position: relative;
+  border: 2px solid #31bb6b;
+}
+
+.userImageUserPost img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  scale: 1.5;
+}
+
+.startPostBtn {
+  width: 100%;
+  border-radius: 25px;
+  background-color: transparent;
+  /* border: 1px solid #acacac; */
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+  outline: none;
+  border: none;
+  color: #000;
+  font-weight: 900;
+}
+
+.startPostBtn:hover {
+  background-color: #00000010;
+  border: 0;
+  color: #000 !important;
+}
+.icons {
+  width: 25px;
+}
+
+.icons svg {
+  stroke: #000;
+}
+
+.icons.dark {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background-color: transparent;
+}
+
+.icons.dark svg {
+  stroke: #000;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`